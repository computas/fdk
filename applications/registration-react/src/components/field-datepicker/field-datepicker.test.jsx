import React from 'react';
import { shallow } from 'enzyme';
import DatepickerField from './field-datepicker.component';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {
    input: { name: 'date', value: '2018-05-15T12:46:34.495Z' },
    showLabel: true,
    label: 'label'
  };
  wrapper = shallow(<DatepickerField {...defaultProps} />);
});

test('should render DatepickerField correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
