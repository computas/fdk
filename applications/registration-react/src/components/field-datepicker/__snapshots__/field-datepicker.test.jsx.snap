// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DatepickerField correctly 1`] = `
<div
  className="pl-2"
>
  <label
    className="fdk-form-label"
    htmlFor="date"
  >
    label
    <DatePicker
      allowSameDay={false}
      className="fdk-reg-datepicker"
      dateFormat="DD.MM.YYYY"
      dateFormatCalendar="MMMM YYYY"
      disabled={false}
      disabledKeyboardNavigation={false}
      dropdownMode="scroll"
      id="date"
      locale="nb-no"
      monthsShown={1}
      onBlur={[Function]}
      onChange={[Function]}
      onClickOutside={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onMonthChange={[Function]}
      onSelect={[Function]}
      onYearChange={[Function]}
      preventOpenOnFocus={false}
      selected={"2018-05-15T12:46:34.495Z"}
      shouldCloseOnSelect={true}
      showTimeSelect={false}
      showYearDropdown={true}
      timeIntervals={30}
      withPortal={false}
      yearDropdownItemNumber={5}
    />
  </label>
</div>
`;
